Ihor/Netflix/src/redux/Sagas/FetchSaga.ts

```diff
@@ -12,11 +9,9 @@ export const fetchMoviesApi = async (
   search?: string
 ): Promise<IMovie[]> => {
   let queryUrl =
-    searchBy && search
-      ? `https://reactjs-cdp.herokuapp.com/movies/?${
-          searchBy === "title" ? `search=${search}` : `filter=${search}`
-        }&searchBy=${searchBy}&sortBy=${sortBy}&sortOrder=desc&offset=${offset}&limit=9`
-      : `https://reactjs-cdp.herokuapp.com/movies?sortBy=${sortBy}&sortOrder=desc&offset=${offset}&limit=9`;
+    searchBy && search ? `https://reactjs-cdp.herokuapp.com/movies/?${searchBy === "title" ? `search=${search}` 
+    : `filter=${search}`}&searchBy=${searchBy}&sortBy=${sortBy}&sortOrder=desc&offset=${offset}&limit=9`
+    : `https://reactjs-cdp.herokuapp.com/movies?sortBy=${sortBy}&sortOrder=desc&offset=${offset}&limit=9`;
```

Comment:
still stink 👎 :)) You can use 
```
queryString.stringify({
   YOUR PARAMS
})
```
It will return you a string that you need without undefind values and then you just add this part to the main